kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java

```diff
@@ -214,7 +214,7 @@ public String getFormattedMessage(String key, Object... args)
 			this.baseName + ", locale=" + getLocale().toString() +
 			", key=" + key + ", pattern=" + pattern);
 		}
-			
+		pattern = pattern.replaceAll("'", "''");
```

Comment:
Why is this escaping needed?

Reply:
Would be "La date d'ouverture..." 

Date is missing... {0}

![image](https://user-images.githubusercontent.com/15141743/188695109-e3c2a40c-64e6-4d31-8e1b-41051be8a6ba.png)

Reply:
I think like this would cause a problem for anyone that has formatted this correctly? Maybe this should just be changed in the message bundle instead with the date. It looks like someone updated that today on Transifex.  

I'm seeing some text like this where it seems like this is escaping correctly. To do it programmatically it _seems_ like you'd need to avoid these already double quoted cases? 

```
calendar-bundles/resources/calendar_mn.properties
181:alert_sitegroupnotdefined=\u0411\u0430\u0439\u0440\u043B\u0430\u043B/\u0431\u04AF\u043B\u044D\u0433 \u0442\u0430\u0439\u043B\u0431\u0430\u0440\u0442\u0430\u0439 ''{0}'' \u043E\u043B\u0434\u0441\u043E\u043D\u0433\u04AF\u0439.
```

Reply:
alert_sitegroupnotdefined shouldn't be changed.